spec/unit/puppet_forge/connection_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'shared/puppet_forge/connection'
+
+describe PuppetForge::Connection, :vcr => false do
```

Comment:
Yep, leftover from integrating the Faraday::Adapter::Test based tests with the existing vcr tests when I tried to use both; I've removed this.